### DISCUSSION TITLE
using System;

### DISCUSSION THREAD

**Discussion by adailson-almeida** [balta-io/2801#69] [Perguntas e Respostas] [answered]
Boa tarde Balta, primeiramente parabéns pela iniciativa, são ótimos os cursos e estou aprendendo bastante. Agora vamos a dúvida. O que tenho que fazer para o arquivo.cs já iniciar com using System; namespace, class Program...instalei o .NET  e o VS Code tudo certinho, mas quando inicio um novo app só vem uma linha "**Console.WriteLine("Hello, World!")**";.

**Comment by andrebaltieri** [comment, marked as answer]
@adayllson o template do .NET 6/C# 10 mudou, então só vem esta linha mesmo, mas todo código do curso funciona perfeitamente...

**Reply by adailson-almeida** [reply]
Obrigado @andrebaltieri!
